crates/typst-svg/src/shape.rs

```diff
@@ -33,7 +32,12 @@ impl SVGRenderer<'_> {
             );
         }
 
-        let path = convert_geometry_to_path(&shape.geometry);
+        // reset transform because we applied it as element's transform attribute
+        // we kept it before for shape paint transform
+        self.xml.write_attribute("transform", &SvgMatrix(state.transform));
+        state = state.with_transform(Transform::identity());
+
+        let path = convert_geometry_to_path(&state.transform, &shape.geometry);
```

Comment:
The transform is always identity at this point, so we could remove this argument, right?